### PR TITLE
Controller: show the number of units added/left

### PR DESCRIPTION
**Pull request by andre8244**
Controllers without a subscription plan can manage up to 3 units. Show a card and a progress bar with the number of unit left.

Other changes:
- Improve description of **Unit manager** page
- Rename `UnitsView` to `UnitManagerView`

See:
- https://github.com/NethServer/nethsecurity/issues/416

### CODE REVIEW

File: src/views/controller/UnitManagerView.vue [hunk 1]
```diff
@@ -143,8 +150,47 @@ function getBulkActionsKebabMenuItems() {
...
+          v-if="defaultsStore.defaultsLoaded && !defaultsStore.validSubscription"
+          :title="
+            t(
+              'controller.units.num_units_left',
+              {
+                num: unitsStore.MAX_NO_SUBSCRIPTION_UNITS - unitsStore.units.length
+              },
+              unitsStore.MAX_NO_SUBSCRIPTION_UNITS - unitsStore.units.length
+            )
+          "
```
[the remark this answers is not in the document]
Comment (andre8244) [reply src/views/controller/UnitManagerView.vue:177]: No, it is used for pluralization

File: src/stores/controller/units.ts
[the remark this answers is not in the document]
Comment (andre8244) [reply src/stores/controller/units.ts:1]: Can't see any line reference. If you are referring to `MAX_NO_SUBSCRIPTION_UNITS`, what's the advantage of having it outside the store?

File: src/views/controller/UnitManagerView.vue (hunk 1 above)
[the remark this answers is not in the document]
Comment (Tbaile) [reply src/views/controller/UnitManagerView.vue:177]: Then all good

File: src/stores/controller/units.ts
[the remark this answers is not in the document]
Comment (Tbaile) [reply src/stores/controller/units.ts:1]: The advantage is avoiding loading it in the application, keeping it static allows to fetch the value without instantiating the store, bit overkill but I'd like the choice

**Review by Tbaile — APPROVED** [review]

**Pull request merged**